.known-issues/doc/duplicate.conf

```diff
@@ -4,3 +4,9 @@
 ^(?P<filename>([\-:\\/\w\.])+[/\\]doc[/\\]api[/\\]io_interfaces.rst):(?P<lineno>[0-9]+): WARNING: Duplicate declaration.
 #
 ^(?P<filename>([\-:\\/\w\.])+[/\\]doc[/\\]subsystems[/\\]sensor.rst):(?P<lineno>[0-9]+): WARNING: Duplicate declaration.
+#
+^(?P<filename>([\-:\\/\w\.])+[/\\]doc[/\\]api[/\\]io_interfaces.rst): WARNING: Duplicate declaration, also defined in \'subsystems/sensor\'\.
```

Comment:
@dbkinder I believe those are not real warnings, but can you please double-check?

Reply:
The actual warnings as they were shown by sphinx are:

```
/home/carles/src/zephyr/doc/api/io_interfaces.rst: WARNING: Duplicate declaration, also defined in 'subsystems/sensor'.
Declaration is 'sensor_value'.

/home/carles/src/zephyr/doc/api/io_interfaces.rst: WARNING: Duplicate declaration, also defined in 'subsystems/sensor'.
Name of declaration is 'sensor_value'.

/home/carles/src/zephyr/zephyr/doc/subsystems/sensor.rst: WARNING: Duplicate declaration, also defined in 'api/io_interfaces'.
Declaration is 'sensor_value'.
/home/carles/src/zephyr/zephyr/doc/subsystems/sensor.rst: WARNING: Duplicate declaration, also defined in 'api/io_interfaces'.
Name of declaration is 'sensor_value'.
```

Reply:
The "duplicate declaration" warning is a known "Sphinx/Breathe" issue as discussed in https://github.com/sphinx-doc/sphinx/issues/4820 and https://github.com/michaeljones/breathe/pull/377  
For now we're adding the pattern to ignore in our ".known_issues" system, sigh.

Reply:
Right, but it's strange that when enabling multicore builds these 2 new ones appeared. Might have something to do with the way sphinx builds.

Reply:
Thanks for the links btw

Reply:
@dbkinder funnily this PR is merged https://github.com/michaeljones/breathe/pull/377. Do we need to update our breath/sphinx versions?